circle.yml

```diff
@@ -0,0 +1,3 @@
+machine:
+  node:
+    version: 6.9.4
```

Comment:
Are we dropping support for older version of node or is this just temporary? I this case, can you add a TODO (if possible).